### DISCUSSION TITLE
Avif support in cbz would be the perfect solution

### DISCUSSION THREAD

**Discussion by angang** [sumatrapdfreader/sumatrapdf#2515] [Ideas]
Hello,

Avif is wthout any doubt the best format for ratio/compression/quality. 
Do you think that it is possible to add support for Sumatra ?
Thx

**Comment by GitHubRulesOK** [comment]
It may be good but is dual purpose and for PDF is of no use in fact there were 2 discussions as to its merits being borderline for documents such as comics but has not been totally dismissed see https://github.com/sumatrapdfreader/sumatrapdf/issues/1520 and https://github.com/sumatrapdfreader/sumatrapdf/discussions/2473

**Reply by angang** [reply]
Thx for your precisions. 
Hope that there will be massive adoption (Waiting for Safari and Edge).

**Comment by kjk** [comment]
For feature requests please open [an issue](https://github.com/sumatrapdfreader/sumatrapdf/issues) so that the status can be tracked.

**Comment by kjk** [comment]
You can vote on this feature at https://sumatrapdf.canny.io/feature-requests/p/support-avif-image-format

**Reply by angang** [reply]
Done, thx Krzysztof !

**Comment by kjk** [comment]
Added .avif support in latest pre-release https://www.sumatrapdfreader.org/prerelease

**Reply by angang** [reply]
Fantastic, it works like a charm !
Thank you kjk  👍
